polaris/tasks/ocean/customizable_viz/viz_horiz_field.py

```diff
@@ -0,0 +1,188 @@
+import cmocean  # noqa: F401
+import numpy as np
+
+from polaris.ocean.model import OceanIOStep
+from polaris.viz import (
+    determine_time_variable,
+    get_viz_defaults,
+    plot_global_mpas_field,
+)
+
+
+class VizHorizField(OceanIOStep):
+    def __init__(self, component, name, indir):
+        super().__init__(component=component, name=name, indir=indir)
+
+    def runtime_setup(self):
+        section = self.config['customizable_viz']
+        self.mesh_file = section.get('mesh_file')
+        self.input_file = section.get('input_file')
+
+    def run(self):  # noqa:C901
+        section_name = 'customizable_viz_horiz_field'
+        section = self.config[section_name]
+
+        # Determine the projection from the config file
+        projection_name = section.get('projection')
+        central_longitude = section.getfloat('central_longitude')
+
+        # Descriptor is none for the first variable and assigned thereafter
+        descriptor = None
+
+        ds_mesh = self.open_model_dataset(
+            self.mesh_file, decode_timedelta=False
+        )
+        min_latitude = section.getfloat('min_latitude')
+        max_latitude = section.getfloat('max_latitude')
+        min_longitude = section.getfloat('min_longitude')
+        max_longitude = section.getfloat('max_longitude')
+        lat_cell = ds_mesh['latCell'] * 180.0 / np.pi
+        lon_cell = ds_mesh['lonCell'] * 180.0 / np.pi
+        if min_longitude < 0.0 and lon_cell.min().values > 0.0:
+            max_longitude_copy = max_longitude
+            max_longitude = 360.0 - min_longitude
+            min_longitude = max_longitude_copy
+        cell_indices = np.where(
+            (ds_mesh.maxLevelCell > 0)
+            & (lat_cell >= min_latitude)
+            & (lat_cell <= max_latitude)
+            & (lon_cell >= min_longitude)
+            & (lon_cell <= max_longitude)
+        )
```

Comment:
This culling process is a little bit risky because cells with centers outside the range may nevertheless have vertices in the range and the boundary may look ragged with this approach.

Also, many projections don't have straight edges in latitude/longitude space so this may drop cells that should appear within the project bounds.  Might it be better to have `mosaic` try to do this kind of optimization and not try to handle it on the Polaris side?